fendermint/actors/eam/src/lib.rs

```diff
@@ -59,6 +68,33 @@ impl IPCEamActor {
 
         Ok(())
     }
+
+    fn update_deployers(rt: &impl Runtime, deployers: Vec<Address>) -> Result<(), ActorError> {
```

Comment:
do we do checks on the deployers? Like simple dedup, size limiting?

Reply:
I think maybe we should restrict to size at least 1 so that we still have someone to update the deployers.

Reply:
oh yeah, it's a map, so no need to do dedup.

Reply:
On 0 check: that would make it impossible to remove all deployment privileges, which is a legitimate use case.